src/main/scala/ai/privado/languageEngine/java/passes/config/PropertiesFilePass.scala

```diff
@@ -102,6 +116,9 @@ class PropertiesFilePass(cpg: Cpg, projectRoot: String) extends ForkJoinParallel
     }
     .l
 
+  private def getMember(member: String) =
+    cpg.member.where(_.name(member)).toList
```

Comment:
It seems we are not checking the member variable with the type from the XML bean ?

Reply:
If you're referring to the class of the variable here, we are filtering it out in the next stage in the `XMLParserBean` function.

Reply:
This method is getting called from line `98` and it is creating the edge between the given member and the property node. I guess the check had to be done before we establish the edge.

Reply:
The current check you are doing is while build the property node. Which guess is not required, as we might be just ok to create the property node (key and value pair) without this check.

Reply:
I checked the flow, and it is filtering out before we are adding the edge, so we are good to go.